README.rst

```diff
@@ -0,0 +1,68 @@
+LTI Consumer XBlock |Build Status| |Coveralls|
+----------------------------------------------
+
+This XBlock implements the consumer side of the LTI specification enabling
+integration of third-party LTI provider tools.
+
+Installation
+------------
+
+Install the requirements into the python virtual environment of your
+``edx-platform`` installation by running the following command from the
+root folder:
+
+.. code:: bash
+
+    $ pip install -r requirements.txt
+
+Enabling in Studio
+------------------
+
+You can enable the LTI Consumer XBlock in Studio through the
+advanced settings.
+
+1. From the main page of a specific course, navigate to
+   ``Settings ->    Advanced Settings`` from the top menu.
+2. Check for the ``advanced_modules`` policy key, and add
+   ``"lti_consumer"`` to the policy value list.
+3. Click the "Save changes" button.
+
+Workbench installation and settings
+-----------------------------------
+
+Install to the workbench's virtualenv by running the following command
+from the xblock-lti-consumer repo root with the workbench's virtualenv activated:
+
+.. code:: bash
+
+    $ make install
+
+Running tests
+-------------
+
+From the xblock-lti-consumer repo root, run the tests with the following command:
+
+.. code:: bash
+
+    $ make test
+
+Running code quality check
+--------------------------
+
+From the xblock-lti-consumer repo root, run the quality checks with the following command:
+
+.. code:: bash
+
+    $ make quality
+
```

Comment:
IMO, the `compile_sass` instructions should be mentioned in here, along with when it's appropriate to run it (i.e. when the SCSS files change).